examples/data_augmentation/config.yml

```diff
@@ -0,0 +1,59 @@
+data:
```

Comment:
Interesting that you use a YAML config to build the pipeline. I like it. Can you give me some feedbacks on the structure of such a configuration. I personally find it less organized sometimes, so would like to hear the feedback from a people who first work on it.